department-of-veterans-affairs/caseflow
# Do not show inactive users as options for task assignee

Resolves #12949

### Description
In "Assign to Person"  task assignment modals, we were offering inactive users 

### Acceptance Criteria
- [ ] Test suite passes.

### Testing Plan
1. Log in as translation team member #1, e.g.  ORG_QUEUE_USER_1
2. Navigate to http://localhost:3000/organizations/translation, open a case (e.g. http://localhost:3000/queue/appeals/2e7d0f5b-f9eb-4767-9b41-40426d9b0562)
3. In task actions, go to "Assign to Person".  You should see Translation team member #5. 
4. Now make Translation team member #5 as inactive in the rails console: 
`User.find_by_css_id("ORG_QUEUE_USER_5").inactive!`
5. Repeat steps 1 through 3.  You should no longer see Team Member 5.

### User Facing Changes
BEFORE: 
<img width="1029" alt="Screen Shot 2020-01-13 at 9 54 00 AM" src="https://user-images.githubusercontent.com/1034221/72265627-ce6b9d80-35ea-11ea-8a78-5fa5fd7e2158.png">

AFTER:
<img width="1070" alt="Screen Shot 2020-01-13 at 9 58 46 AM" src="https://user-images.githubusercontent.com/1034221/72265913-4f2a9980-35eb-11ea-846f-a4242d69edf1.png">
